.github/dependabot.yml

```diff
@@ -62,4 +62,15 @@ updates:
     labels:
```

Comment:
This file doesn't matter except in 'main' branch but I wanted to keep these changes together